library/common/2105.0.0/templates/lib/containers/_port.tpl

```diff
@@ -0,0 +1,13 @@
+{{/*
+Retrieve ports configuration for container
+*/}}
+{{- define "common.containers.configurePorts" -}}
+ports:
```

Comment:
If `ports` is empty list, this would still render. Let's please have this under a conditional to avoid that case.